rh/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+from users.models import JustfixUser
+from project.util import phone_number as pn
+from project.util.address_form_fields import ADDRESS_FIELD_KWARGS, BOROUGH_FIELD_KWARGS
+
+
+class RentalHistoryRequest(models.Model):
```

Comment:
@sraby, I think you are probably already familiar with Django Models from the Django tutorial you went through a while ago, but let me know if not!